README.md

```diff
@@ -1,4 +1,4 @@
-# UpCloud CSI Driver ![GitHub Actions status](https://github.com/UpCloudLtd/upcloud-csi/actions/workflows/deploy.yml/badge.svg)
+# UpCloud CSI Driver ![GitHub Actions status](https://github.com/UpCloudLtd/upcloud-csi/actions/workflows/deploy.yaml/badge.svg)
 
```

Comment:
Could also move the badge from header to next row to be more in sync with CLI and SDK repos

Reply:
Good point, I'll change that.